base/loading.jl

```diff
@@ -3548,10 +3548,10 @@ julia> cd("/home/JuliaUser") # working directory
 
 julia> # create script at /home/JuliaUser/Projects
        open("/home/JuliaUser/Projects/test.jl","w") do io
-           print(io, """
+           print(io, \"\"\"
```

Comment:
Note that it is also sufficient to only escape any one of the `"`, if that feels better. There is no style preference either way.
```suggestion
           print(io, \"""
```